packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6088,6 +6088,17 @@ export default class Meeting extends StatelessWebexPlugin {
         body.resourceToken = resourceToken;
       }
 
+      // @ts-ignore
+      this.webex.internal.newMetrics.submitClientEvent({
```

Comment:
just to double check: are we supposed to send this to CA before we are actually making the request and at that point we don't know if that request succeeded or not and I don't see anything being sent from the catch clause

Reply:
To my understanding, this metric specifically marks the instance that a request is sent, but doesn't indicate the success or failure of it. Something should probably be sent in the catch, but whiteboard failures are different from share failures. To make this more clear I will remove this metric for now since we were planning on handling whiteboard in a different epic.